Automattic/sensei
# Move the email footer to the page template

Fix #6714 

## Proposed Changes
* Remove the email-footer pattern because it is not loading properly when the user tries to edit the email template.

## Context
Currently,  the email footer is not available when we are loading the email template, probably due to the load sequence, so I moved to the email template until we find a better solution. 

## Testing Instructions
- Go to Sensei > Settings > E-mails 
- Click "Edit" on  any email
- Template "Sensei Email"
- Check if the footer is loaded properly. 

## Pre-Merge Checklist
